res/css/views/rooms/wysiwyg_composer/components/_FormattingButtons.pcss

```diff
@@ -92,6 +92,10 @@ limitations under the License.
     .mx_FormattingButtons_Button_strikethrough::before {
         mask-image: url('$(res)/img/element-icons/room/composer/strikethrough.svg');
     }
+
+    .mx_FormattingButtons_Button_inline_code::before {
+        mask-image: url('$(res)/img/element-icons/room/composer/inline_code.svg');
```

Comment:
Just realised, that we're using that CSS here, but we should use SVG icons with `currentColor` rather than the `mask-image` technic.

See https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/icons.md

Reply:
OK, I'll change it in a dedicated PR.